### ISSUE TITLE
HomeScreen screen buttons should be aria-describedby their `p` description content

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#648]
From https://github.com/phetsims/joist/issues/496 while reviewing HomeScreen.md

**Comment by zepumph** [comment]
Oops! I now see that this is already implemented, I got confused because it isn't needed for the nav bar buttons, only on the HomeScreen, and I didn't see them on the nav bar buttons. Closing
